src/index.scss

```diff
@@ -1,5 +1,5 @@
-@import "~normalize-scss/sass/normalize/import-now";
-@import "~bootstrap/scss/bootstrap";
-@import "./components/ToastDrawer/ToastDrawer";
-@import "./components/LazyLoaderFallback/LazyLoaderFallback";
-@import "./style.scss";
+@import '~normalize-scss/sass/normalize/import-now';
```

Comment:
Looks like we should add an override to `.prettierrc` that sets doubleQuote to true for .scss files...

Reply:
This might've just been something weird on my end, some settings in VSCode or something. I'll update this.